samples/Network/deny-nsg-inbound-traffic-change/README.md

```diff
@@ -0,0 +1,26 @@
+# Denies NSG rule changes that allow all inbound traffic
```

Comment:
Nit: I think the folder/name should reflect that the policy is specifically denying  security rules which "Allow All (*)", rather than all inbound rules. Something like `deny-nsg-inbound-allow-all` ?